docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -1115,63 +1115,165 @@ For the purposes of our simulation, we made a few assumptions:
+    intervention *if* they have been eating food for at least 6 months; that is,
+    they receive the full effect if they are at least one year old, or
+
+    b) If they are less than one year old, they have started or will start
+    eating fortified food as soon as they turn(ed) 6 months old, hence will
+    achieve the full age-dependent effect at exactly one year old.
+
+To summarize, we are trying to account for two factors that will decrease the
+reported effect size of iron fortification on hemoglobin in our simulants:
+
+1.  (**Age effect**) Children don't start eating (fortified) solid food until 6
+    months of age, at which point the amount of (fortified) food slowly
+    increases to a maximum at 2 years of age, then remains constant.
+
+2.  (**Time-lag effect**) When someone starts eating iron-fortified food, the
+    measured effect on hemoglobin increases from 0 to the maximum effect size
+    over a period of 6 months.
+
+To model these two effects mathematically, we are making the following
+simplifying assumptions:
+
+1.  The above two factors each reduce the reported Hb effect size by a multiplicative factor independent of the other factor.
+
+2.  Each multiplicative factor is a continuous piecewise linear function of time.
+
+We will model the age effect with a continuous piecewise linear function
+``hb_age_fraction(age)``, where ``age`` is the current age of the simulant, and
+we will model the time-lag effect with a continuous piecewise linear function
+``hb_lag_fraction(time_since_fortified)``, where ``time_since_fortified`` is the
+time since the simulant started eating iron-fortified food:
 
 .. code-block:: Python
 
-	# at the draw level
-	MD = hb_md_distribution.rvs() # mean difference in hemoglobin concentration due to iron fortification
-		# note, hb_md_distribution defined above in the effect size section
+  def hb_age_fraction(age):
+  """
+  Multiplier on Hb effect size due to children eating less food at younger
+  ages. (`age` is current age in years).
+  """
+    return 0 if age<0.5 else (age-0.5)/1.5 if age<2 else 1
+
+  def hb_lag_fraction(time_since_fortified):
+  """
+  Multiplier on Hb effect size due to lag in response time to iron fortification.
+  The argument `time_since_fortified` is the time (in years) since a simulant
+  started eating fortified food (note that a negative value of `time_since_fortified`
+  indicates the child has not yet started eating fortified food).
+  """
+    return (0                         if time_since_fortified < 0   else
+            time_since_fortified/0.5  if time_since_fortified < 0.5 else
+            1)
+
+.. Note::
+
+  Since children don't start eating fortified food until 6 months of age, the
+  argument ``time_since_fortified`` for the time-lag function will be *either*
+  the time since the child's household got fortification coverage, *or* the time
+  since the child turned 6 months old, *whichever is smaller*.
```

Comment:
Is this consistent with the similar comment in the code block below?

> # Fortification starts either when the household receives coverage, or when
> # time_since_coverage_i	    # the child turns 6 months old and starts eating solids, whichever is later.
>

Reply:
Yes, I think so: Later time = smaller duration of time since it happened